features/facebookImport/test/ministory/json-file-names-analysis.test.js

```diff
@@ -11,12 +11,13 @@ const commonJsonFiles = commonStructure
     });
 
 async function analyzeZipWithFiles(files) {
-    const zipFile = new ZipFileMock();
+    let dataFilePairs = [];
```

Comment:
Nit: This should still be possible to be const, since it's not being reassigned.